core/scripts/vrfv2/testnet/super_scripts.go

```diff
@@ -29,6 +64,8 @@ func deployUniverse(e helpers.Environment) {
 	// optional flags
 	fallbackWeiPerUnitLinkString := deployCmd.String("fallback-wei-per-unit-link", assets.GWei(60_000_000).String(), "fallback wei/link ratio")
 	registerKeyUncompressedPubKey := deployCmd.String("uncompressed-pub-key", "", "uncompressed public key")
+	registerKeyCompressedPubKey := deployCmd.String("compressed-pub-key", "<COMPRESSED_VRF_KEY>", "compressed public key")
+	registerKeyHash := deployCmd.String("key-hash", "<KEY_HASH>", "key hash")
```

Comment:
You can generate these values based on the uncompressed pubkey, so that you only ever have to provide uncompressed:

```golang
pubBytes, err := hex.DecodeString(uncompressed)
helpers.PanicErr(err)
pk, err := crypto.UnmarshalPubkey(pubBytes)
helpers.PanicErr(err)
compressedPk := crypto.CompressPubkey(pk)
keyHash := crypto.Keccak256Hash(compressed)
```

Reply:
Nice! Yeah I figured but I couldn't find anywhere in code that being done.